frontend/src/routes/SendConfirm.tsx

```diff
@@ -41,6 +41,7 @@ function SendConfirm() {
 
         if (paymentType === PaymentType.onchain) {
           const txid = await nodeManager?.send_to_address(destination, amountInt);
+          await nodeManager?.sync();
           setTxid(txid)
           setSentOnchain(true);
```

Comment:
I think it'd be better to sync after we set the UI so the user doesn't have to wait for it